config/test.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
       username: 'test',
       password: 'tset',
     },
+    notificationEmail: '',
```

Comment:
Is it worth putting a dummy email address in here? Just so we see an email being sent from our mail client to the right address?

Reply:
Yep. Amended.